docs/book/src/clusterctl/provider-contract.md

```diff
@@ -236,35 +236,37 @@ easier transition from `kubectl apply` to `clusterctl`.
 As a reference you can consider the labels applied to the following
 providers.
 
-| Provider Name| Label                                                 |
-|--------------|-------------------------------------------------------|
-|CAPI          | cluster.x-k8s.io/provider=cluster-api                 |
-|CABPK         | cluster.x-k8s.io/provider=bootstrap-kubeadm           |
-|CABPM         | cluster.x-k8s.io/provider=bootstrap-microk8s          |
-|CACPK         | cluster.x-k8s.io/provider=control-plane-kubeadm       |
-|CACPM         | cluster.x-k8s.io/provider=control-plane-microk8s      |
-|CACPN         | cluster.x-k8s.io/provider=control-plane-nested        |
-|CAPA          | cluster.x-k8s.io/provider=infrastructure-aws          |
-|CAPB          | cluster.x-k8s.io/provider=infrastructure-byoh         |
-|CAPC          | cluster.x-k8s.io/provider=infrastructure-cloudstack   |
-|CAPD          | cluster.x-k8s.io/provider=infrastructure-docker       |
-|CAPDO         | cluster.x-k8s.io/provider=infrastructure-digitalocean |
-|CAPG          | cluster.x-k8s.io/provider=infrastructure-gcp          |
-|CAPH          | cluster.x-k8s.io/provider=infrastructure-hetzner      |
-|CAPIBM        | cluster.x-k8s.io/provider=infrastructure-ibmcloud     |
-|CAPKK         | cluster.x-k8s.io/provider=infrastructure-kubekey      |
-|CAPK          | cluster.x-k8s.io/provider=infrastructure-kubevirt     |
-|CAPM3         | cluster.x-k8s.io/provider=infrastructure-metal3       |
-|CAPN          | cluster.x-k8s.io/provider=infrastructure-nested       |
-|CAPO          | cluster.x-k8s.io/provider=infrastructure-openstack    |
-|CAPOCI        | cluster.x-k8s.io/provider=infrastructure-oci          |
-|CAPP          | cluster.x-k8s.io/provider=infrastructure-packet       |
-|CAPV          | cluster.x-k8s.io/provider=infrastructure-vsphere      |
-|CAPVC         | cluster.x-k8s.io/provider=infrastructure-vcluster     |
-|CAPVCD        | cluster.x-k8s.io/provider=infrastructure-vcd          |
-|CAPX          | cluster.x-k8s.io/provider=infrastructure-nutanix      |
-|CAPZ          | cluster.x-k8s.io/provider=infrastructure-azure        |
-|CAPOSC        | cluster.x-k8s.io/provider=infrastructure-outscale     |
+| Provider Name | Label                                                 |
+|---------------|-------------------------------------------------------|
+| CAPI          | cluster.x-k8s.io/provider=cluster-api                 |
+| CABPK         | cluster.x-k8s.io/provider=bootstrap-kubeadm           |
+| CABPM         | cluster.x-k8s.io/provider=bootstrap-microk8s          |
+| CABPKK3S      | cluster.x-k8s.io/provider=bootstrap-kubekey-k3s       |
```

Comment:
@24sama 
Just for confirmation (no objection from my side);

CABPKK3s is intentional? vs CABPKKK3s   

("kubekey 3s" vs "kubekey k3s")

Reply:
Ah sorry. KKK is pretty much a no-go given history..